spec/controllers/v2/test_results_controller_spec.rb

```diff
@@ -72,6 +73,14 @@
         end.sort_by(&:id)
       end
 
+      let(:rule_result_data) do
+        %w(low medium high).each_with_object({}) do |level, obj|
+          obj["#{level}_severity_failing_rule".to_sym] = pw({
+            Faker::Internet.uuid => [level, 'fail']
```

Comment:
AFAIK you don't need the left side, so you can get rid of these objects completely. SImply define everything in the YAML.